src/btree/bt_curnext.c

```diff
@@ -517,9 +517,10 @@ __cursor_key_order_check_col(WT_SESSION_IMPL *session, WT_CURSOR_BTREE *cbt, boo
 
     WT_RET(__wt_msg(session, "dumping the tree"));
     WT_WITH_BTREE(session, btree, ret = __wt_debug_tree_all(session, NULL, NULL, NULL));
-    WT_ERR_PANIC(session, EINVAL,
+    __wt_verbose_error(session, WT_VERB_OUT_OF_ORDER,
       "WT_CURSOR.%s out-of-order returns: returned key %" PRIu64 " then key %" PRIu64,
       next ? "next" : "prev", cbt->lastrecno, cbt->recno);
+    WT_ERR_PANIC(session, EINVAL, "found key out-of-order returns");
```

Comment:
We could keep the message used in `__wt_verbose_error`, or keep `found key out-of-order.` (removing `returns`)